added horde map + more

Zasig opened the pull request:
Found this because of Nobbles Video. Nice idea and i wanted to expand it and give something back:

added horde map
added map switch
added plot numbers (with on/off switch)
added plot pictures feature (removed the picture folders for copyright reasons)
-> create 'plots/alliance' and 'plots/horde' in root directory. Put pictures named '0.jpg' '1.jpg' ...etc.
added previews
added table beneath the map
added locale switch
added pw for basic safety (pw to set under config.php)
added panning + zooming

might have forgotten 1-2 things. 

everything made with github copilot

Zasig committed:
initial

Zasig committed:
fix housing number when zooming

Zasig committed:
Delete housing/plots/alliance directory

for copyright reasons

Zasig committed:
Delete housing/plots/horde directory

for copyright reasons

Zasig commented:
Screenshot for preview :)
<img width="2066" height="1315" alt="image" src="https://github.com/user-attachments/assets/21c8b2b5-3917-4e35-b80f-ce5f2708a377" />

Zasig committed:
fixed double housing numbers on horde map

Zasig committed:
Merge branch 'main' of https://github.com/Zasig/wow-housing

Zasig committed:
quick edit 

quick and dirty edit because of double plot numbers on horde map

LudiusMaximus commented:
I cannot find the plot pictures plots/${faction}/${position.plotNumber}.jpg
Are they missing from your repository on purpose?

Zasig commented:
I removed them because the ones i used, where fetched from an external side. since i didn't took them ingame .... i'm no expert in copyright so i just removed them

You can find them still in my old commit on my repo: https://github.com/Zasig/wow-housing/tree/f37979459c35d98c366e2e56909e7ebdd780c3ad/housing/plots

Zasig committed:
Update README to reflect Horde map inclusion
